src/api/v2/account_activities.rs

```diff
@@ -285,10 +281,10 @@ impl Activity {
   }
 
   /// The time at which the activity occurred.
-  pub fn time(&self) -> SystemTime {
+  pub fn time(&self) -> DateTime<Utc> {
```

Comment:
This could be `&DateTime<Utc>` if we can get both variants to store a `DateTime<Utc>` natively (instead of `DateTime<Utc>` and `NaiveData`).

Reply:
Sounds good.